libsrc/osic1p/extzp.s

```diff
@@ -0,0 +1,21 @@
+;
+; Additional zero page locations for the Challenger 1P.
+; NOTE: The zeropage locations contained in this file get initialized
+; in the startup code, so if you change anything here, be sure to check
+; not only the linker config, but also the startup file.
+;
+
+; ------------------------------------------------------------------------
+
+        .include        "extzp.inc"
+
+.segment        "EXTZP" : zeropage
+
+CURS_X:         .byte   0
+CURS_Y:         .byte   0
+CURS_SAV:       .byte   0
+SCR_LINELEN:    .byte   24
```

Comment:
The `.byte <value>`directive is only useful in segments that get written (like `TEXT`, `DATA`, `RODATA`). But in bss-type segments (like `BSS`, `ZP`, `EXTZP`) the value is lost anyway. So I suggest to use only the `.res` directive for definitions in bss-type segments.